src/main/java/hudson/plugins/copyartifact/CopyArtifact.java

```diff
@@ -124,7 +126,14 @@ public CopyArtifact(String projectName, String parameters, BuildSelector selecto
 
         this.project = projectName;
         this.parameters = Util.fixEmptyAndTrim(parameters);
-        this.selector = selector;
+
+        if (selector == null) {
+            // TODO: specify the BuildSelector instance in the workflow script via CoreStep?
+            this.selector = new StatusBuildSelector(true);
```

Comment:
Thanks, will do.

Also wondering how would users specify different `BuildSelector` types  from within the workflow script?